public/app/features/dashboard/components/ShareModal/utils.ts

```diff
@@ -43,7 +43,7 @@ export function buildParams({
 }
 
 export function buildBaseUrl() {
-  let baseUrl = window.location.href;
+  let baseUrl = window.parent.location.href;
```

Comment:
Should this change have a comment saying that it's to hide `dashboardhost` from the user? I could easily mistake this for an upstream change and blow it away by accident when upgrading to a new version of Grafana.

Reply:
Yes - comment added.